server/views/pages/overview/partials/educationAndTrainingTab/educationAndTrainingTabContents.njk

```diff
@@ -16,7 +16,7 @@ The 'Training and eduction in prison' summary card is only shown if the short qu
   </section>
 {% else %}
   <section data-qa="qualifications-and-education-history-short-question-set">
-    {% include './_qualificationsAndEducationHistory_inductionShortQuestionSet.njk' %}
+    {% include './_inductionShortQuestionSet.njk' %}
```

Comment:
Can we rename this back please? I think the full name is better as it matches the corresponding file for the long journey, and it describes that it contains the Qualifications And Education History content (as opposed any other content for long vs. short question set)

Reply:
Fixed